shelley/design-spec/delegation_design_spec.tex

```diff
@@ -699,21 +745,25 @@ \subsubsection{Reward Account Address}
 \item They use account-style accounting, not UTxO-style.
 \item They can not receive funds via transactions. Instead, their
   balance is only increased when rewards are distributed.
-\item They are not associated with a spending key. Instead, there is a
-  one-to-one correspondence between registered staking keys and reward
+\item They do not contain a value address. Instead, there is a
+  one-to-one correspondence between registered stake addresses and reward
   account addresses (see \cref{fig:er-relationship-delegation}). Whenever funds
-  are withdrawn from a reward account address, its staking key is used in the
-  witness.
+  are withdrawn from a reward account address, a witness for its stake address
+  has to be provided.
 \end{itemize}
 
-A reward address has the form
+A reward account address has the form
 \[
-\hash{vks} \mathbin{||} \beta
+\mathcal{A}_s \mathbin{||} \beta
 \]
-where \(\hash{vks}\) is a cryptographic hash of the public staking key
-of the address. This key is used whenever funds are withdrawn from the
-address. Furthermore, stake associated with funds in the address
-contributes to the stake of this key.
+where \(\mathcal{A}_s\) is a stake address\footnote{Strictly speaking, this is a
+deviation from the general form of an address in \cref{eq:address-general},
+where the first part of an address is always a value address, not a stake
+address. For reward account addresses, \(\mathcal{A}_s\) takes the part of both
+the value address and the stake address.}. Withdrawing funds from the reward
+account address requires a witness for this stake address. Furthermore, stake
+associated with funds in the address contribute to the stake of this stake
+address.
```

Comment:
I'm confused! This makes it sound like a different thing, but isn't this exactly a stake address, nothing more nothing less? There's no beta.

Reply:
In my head, there's a distinction between

- an address, which can hold funds and optionally stake
- a value address, which is part of an address
- a stake address, which is also part of an address.

So, the reward account address would be a special case of an address, that is determined solely by a stake address (which in this case controls both value and stake). Kind of like a lifted version, since a stake address on its own can't hold value.

But maybe this distinction is not needed?